org.eclipse.lsp4j.debug/src/main/java/org/eclipse/lsp4j/debug/DebugProtocol.xtend

```diff
@@ -2627,6 +2627,15 @@ class Capabilities {
 	 * Since 1.51
 	 */
 	Boolean supportsSingleThreadExecutionRequests;
+	
+	/**
+  	 * Client supports the `startDebugging` request.
+  	 * <p>
+	 * This is an optional property.
+	 * <p>
+	 * Since 1.59
+	 */
+	Boolean supportsStartDebuggingRequest;
```

Comment:
This is in the wrong place, should be in `InitializeRequestArguments` as it is a capability of the client, not of the server.